### ISSUE TITLE
trailing-comma-tuple not reported for return statements

### ISSUE THREAD

**Issue by bart2** [PyCQA/pylint#2269]
### Steps to reproduce
1. Create a file containing `return 1,`:
```
cat <<EOF > file.py
#!/usr/bin/env python
"""some module"""

def some_func():
    """some function"""

    return 1,
EOF
```
2. Run `pylint` on that file:
```
pylint file.py
```

### Current behavior
```
No config file found, using default configuration

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

### Expected behavior
`trailing-comma-tuple` is reported for the line containing `return 1,`

### pylint --version output
```
No config file found, using default configuration
pylint 1.9.2,
astroid 1.6.5
Python 3.6.3rc1+ (default, May 30 2018, 17:32:30)
[GCC 5.x]
```
